app/retail/templates/home/index2020.html

```diff
@@ -121,26 +123,40 @@ <h1>
   <!-- what is gitcoin -->
 
   <section id="whatisgitcoin">
-    <div class="grid">
-      <div>
-        <h2>Here to earn?</h2>
-        <p>
-          Build exciting projects and upskill on development while earning
-          Ethereum and other tokens. Find mentorship by connecting with
-          talented developers and leaders of Ethereum’s top companies.
-        </p>
-        <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
-        <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
-        <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-8">
+          <h2>Here to Earn?</h2>
+          <p>
+            Build exciting projects and upskill on development while earning
+            Ethereum and other tokens. Find mentorship by connecting with
+            talented developers and leaders of Ethereum’s top companies.
+          </p>
+          <div class="actions">
+          <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+          <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
+          <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+          </div>
+        </div>
+        <div class="col-md-4">
+          <h2>Here to Support?</h2>
+          <p>
+            Discover great open source projects working hard to impact the
+            ecosystem. Get involved by funding, supporting, or mentoring
+            projects to help push the Ethereum forward.
+          </p>
+          <a class="btn fe-pulse actions" href="/grants"><span>Support Grant Projects</span></a>
+        </div>
       </div>
-      <div>
-        <h2>Here to Support?</h2>
-        <p>
-          Discover great open source projects working hard to impact the
-          ecosystem. Get involved by funding, supporting, or mentoring
-          projects to help push the Ethereum forward.
-        </p>
-        <a class="btn fe-pulse" href="/grants"><span>Support Grant Projects</span></a>
+      <div class="row actions-reverse">
+        <div class="col-md-8">
+          <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+          <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
+          <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+        </div>
```

Comment:
@frankchen07 is this intended ? we show the same actions for 
Here to earn and Here to support ?